Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5015,7 +5015,7 @@ PreferFasterClickOverDoubleTap:
 
 PreferPageRenderingUpdatesNear60FPSEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
@smfr: Could you confirm that this status change is accurate? It reflects that the feature is always enabled on Mac.

Reply:
The unusual platform here is visionOS. Again, saying this is "stable" in a platform-independent way may not be right for something we can’t turn on for visionOS. This is a strategic question, I think.

Maybe we’re saying that feature status is mostly about macOS?